### ISSUE TITLE
Compacting script for multiple devices doesnt work for ipv6

### ISSUE THREAD

**Issue by selldinesh** [open-traffic-generator/snappi-ixnetwork#434]
Tried the below snippet to create 1000 IPv4 and 1000 IPv6 devices

    num_of_devices=1000
    conf_values['server_1_ipv4']=get_ip_addresses("192.168.1.2", 2000)[::2]
    conf_values['server_2_ipv4']=get_ip_addresses("192.168.1.2", 2000)[1::2]
    conf_values['server_1_ipv6']=get_ip_addresses("5000::2", 2000,'ipv6')[::2]
    conf_values['server_2_ipv6']=get_ip_addresses("5000::2", 2000,'ipv6')[1::2]
    conf_values['server_1_mac']=get_macs("001600000011",num_of_devices)
    conf_values['server_2_mac']=get_macs("001600000011",num_of_devices)
    import pdb;pdb.set_trace()
    for i in range(1, num_of_devices+1):
        #server1
        d1 = config.devices.device(name='Server_1_{}'.format(i-1))[-1]
        d1.container_name = p3.name
        eth_1 = d1.ethernet
        eth_1.name = 'Ethernet 1_{}'.format(i-1)
        eth_1.mac = conf_values['server_1_mac'][i-1]
        ipv4_1 = d1.ethernet.ipv4
        ipv4_1.name = 'IPv4 1_{}'.format(i-1)
        ipv4_1.address = conf_values['server_1_ipv4'][i-1]
        ipv4_1.gateway = '192.168.1.1'
        ipv4_1.prefix = 16
        #server2
        d2 = config.devices.device(name='Server_2_{}'.format(i-1))[-1]
        d2.container_name = p2.name
        eth_2 = d2.ethernet
        eth_2.name = 'Ethernet 2_{}'.format(i-1)
        eth_2.mac = conf_values['server_2_mac'][i-1]

```
        ipv6_2 = d2.ethernet.ipv6
        ipv6_2.name = 'IPv6 2_{}'.format(i-1)
        ipv6_2.address = conf_values['server_2_ipv6'][i-1]
        ipv6_2.gateway = '5001::1'
        ipv6_2.prefix = 64
```

The moment ipv6 device creation part gets executed , im getting the below error and IPv6 devices are not getting creating whereas IPv4 works perfectly fine

E           SnappiIxnException:   File "/usr/local/lib/python2.7/dist-packages/snappi_ixnetwork/snappi_convergence_api.py", line 51, in set_config
E               self._api.config_ixnetwork(config)
E             File "/usr/local/lib/python2.7/dist-packages/snappi_ixnetwork/snappi_api.py", line 316, in config_ixnetwork
E               self.ngpf.config()
E             File "/usr/local/lib/python2.7/dist-packages/snappi_ixnetwork/ngpf.py", line 44, in config
E               self._api._topology, self._api.snappi_config.devices
E             File "/usr/local/lib/python2.7/dist-packages/snappi_ixnetwork/ngpf.py", line 124, in _configure_topology
E               ixn_topology.DeviceGroup, device, multiplier
E             File "/usr/local/lib/python2.7/dist-packages/snappi_ixnetwork/ngpf.py", line 142, in _configure_device_group
E               self._config_proto_stack(ixn_device_group, device, ixn_device_group)
E             File "/usr/local/lib/python2.7/dist-packages/snappi_ixnetwork/ngpf.py", line 164, in _config_proto_stack
E               self._config_proto_stack(new_ixn_obj, child, ixn_dg)
E             File "/usr/local/lib/python2.7/dist-packages/snappi_ixnetwork/ngpf.py", line 163, in _config_proto_stack
E               new_ixn_obj = stack_class(ixn_obj, child, ixn_dg)
E             File "/usr/local/lib/python2.7/dist-packages/snappi_ixnetwork/ngpf.py", line 370, in _configure_ipv6
E               ixn_ipv6.find(Name="^%s$" % self._api.special_char(ipv6.name))
E            **'dict' object has no attribute 'name'**

**Comment by selldinesh** [comment]
![image](https://user-images.githubusercontent.com/40899231/131631576-1df78deb-8bd6-4c4b-9451-8e52b625c314.png)

**Comment by selldinesh** [comment]
In the above image , for Server_2_0 topology ipv6 doesnt get added, but in server_1_0 ipv4 got created as expected

**Comment by ANISH-GOTTAPU** [comment]
addressed in snappi[ixnetwork]==0.5.3
